src/app/mosh_client.html

```diff
@@ -73,6 +73,10 @@
             <td>Mosh server command:</td>
             <td><input id="server-command" type="text" disabled placeholder="default"></td>
           </tr>
+          <tr id="mosh-escape-key-row">
+            <td>Mosh escape key (default: <code>^</code>)</td>
```

Comment:
To keep the UI clean, leave out the parenthetical defining the default. And besides, the default is Ctrl-^, not ^.